Cesium.Preprocessor.Tests/ExpressionTestExtensions.cs

```diff
@@ -0,0 +1,14 @@
+namespace Cesium.Preprocessor.Tests;
+
+internal static class ExpressionTestExtensions
+{
+    public static bool ToBoolean(this string text)
+    {
+        return text switch
+        {
+            "1" => true,
+            "0" => false,
+            _ => throw new NotSupportedException()
+        };
+    }
```

Comment:
_(Note)_ Just a note that normally `"0"` is considered as falsy, while any other value is considered truthy in C. I have to admit I didn't read the corresponding section on C preprocessor in detail, and maybe we want to do strict checks in our tests.

Reply:
Fact, my mistake, sorry :smile:

Reply:
Oh, no, I've revised the code, it's just an extension class for ease of testing, this method is not used for calculations in C, but simply translates the result, which is always 0 or 1

Reply:
By the way, see `Cesium.Preprocessor.MacroExtensions.AsBoolean`. Consider reusing that one.